package.json

```diff
@@ -79,7 +79,12 @@
   "bin": {
     "nunjucks-precompile": "./bin/precompile"
   },
-  "main": "index",
+  "main": "index.js",
+  "files": [
+    "bin/**",
+    "browser/**",
+    "src/**"
```

Comment:
This also removes the files in `nunjucks/src` from the package. In theory that should be safe, but it would be a breaking change (if somehow, somebody was importing those files). To err on the side of caution we should probably continue to ship them in the package, at least until the next minor version release.

Reply:
That was intended change. This files was never officially provided by the package, if someone hack into package content, he definitely know about risks. I think we could safely make this change, this will save all other users a lot.

Reply:
Also, same files in `src` dir

Reply:
They are not exactly the same files, because they're transpiled, but you make a decent point. If we get a bunch of error reports we can always put out another patch release that adds it back in.